test/e2e/flowaggregator_test.go

```diff
@@ -867,8 +867,9 @@ func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, isDstService
 			for _, record := range recordSlices {
 				flowStartTime := int64(getUint64FieldFromRecord(t, record, "flowStartSeconds"))
 				exportTime := int64(getUint64FieldFromRecord(t, record, "flowEndSeconds"))
+				flowEndReason := int64(getUint64FieldFromRecord(t, record, "flowEndReason"))
 				if strings.Contains(record, src) && strings.Contains(record, dst) && strings.Contains(record, srcPort) {
-					if exportTime >= flowStartTime+iperfTimeSec {
+					if exportTime >= flowStartTime+iperfTimeSec || flowEndReason == 3 {
```

Comment:
same as above, would be good to have a code comment here to remind the reader of what `flowEndReason== 3` means

Reply:
Thanks, added.